features/_your_results/salary_below_minimum_threshold.feature

```diff
@@ -0,0 +1,38 @@
+  In order to understand that there is no requirement for employer contributions to increase over time
+  As an employee earning under £5876
+  I need to see employer contributions stay static
+
+  Scenario Outline: For monthly, 4-weekly and weekly salary frequencies
+    Given my "<salary>" "<salary_frequency>" is below the minimum threshold
+    And my employee contribution is "<my_contribution>"
+    And my employer contribution is "<employer_contribution>"
+    When I move on to the results page
+    Then I should see my employer contributions for current period as "<employer_current_period>"
+    And I should see my employer contributions for second period as "<employer_second_period>"
+    And I should see my employer contributions for second period as "<employer_third_period>"
+
+    Examples:
+        | salary | salary_frequency | my_contribution | employer_contribution | employer_current_period | employer_second_period | employer_third_period |
+        | 416.67 | per Month        | 1               | 0                     | £0.00                   | £0.00                  | £0.00                 |
+        | 416.67 | per Month        | 1               | 1                     | £4.16                   | £4.16                  | £4.16                 |
+        | 384.62 | per 4 weeks      | 1               | 0                     | £0.00                   | £0.00                  | £0.00                 |
+        | 384.62 | per 4 weeks      | 1               | 1                     | £3.84                   | £3.84                  | £3.84                 |
+        | 96.15  | per Week         | 1               | 0                     | £0.00                   | £0.00                  | £0.00                 |
+        | 96.15  | per Week         | 1               | 1                     | £0.96                   | £0.96                  | £0.96                 |
+
+  Scenario Outline: For annual salary frequency
+    Given my "<salary>" "<salary_frequency>" is below the minimum threshold
+    And my employee contribution is "<my_contribution>"
+    And my employer contribution is "<employer_contribution>"
+    When I move on to the results page
+    And I select "<salary_frequency>" to change the calculations
+    And I press recalculate
+    Then I should see my employer contributions for current period as "<employer_current_period>"
+    And I should see my employer contributions for second period as "<employer_second_period>"
+    And I should see my employer contributions for second period as "<employer_third_period>"
+
+    Examples:
+        | salary | salary_frequency | my_contribution | employer_contribution | employer_current_period | employer_second_period | employer_third_period |
+        | 5000   | per Year         | 1               | 0                     | £0.00                   | £0.00                  | £0.00                 |
```

Comment:
Same as above. 
you can also move `salary`, `salary_frequency`, `my_contribution` out of the examples table as these don't change between scenarios.